src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java

```diff
@@ -93,7 +94,7 @@ public Tenant save(UUID consortiumId, UUID adminUserId, Tenant tenantDto) {
     User shadowAdminUser = userService.prepareShadowUser(adminUserId, currentTenantContext.getTenantId());
     userTenantRepository.save(createUserTenantEntity(consortiumId, shadowAdminUser, tenantDto));
 
-    try (var context = new FolioExecutionContextSetter(prepareContextForTenant(tenantDto.getId(), folioModuleMetadata, currentTenantContext))) {
+    try (var context = new FolioExecutionContextSetter(contextHelper.getFolioExecutionContext(tenantDto.getId()))) {
```

Comment:
it is not very clear that we using system user now, I propose to rename this method:
getSystemUserFolioExecutionContext

Reply:
Fixed